### ISSUE TITLE
Investigate StackBlitz issue with `@use`

### ISSUE THREAD

**Issue by trimox** [trimox/angular-mdc-web#2212]
Appears StackBlitz template(s) do not recognize `@use` is a valid Sass rule. Maybe time to look @ alternatives..

StackBlitz template (using canary): https://stackblitz.com/edit/angular-mdc

**Comment by RikEskes** [comment]
It isn't working in my [StackBlitz example](https://stackblitz.com/edit/angular-mdc-kwwzgt) from issue #2189, but perhaps the solution might push you/someone in the right direction?

**Comment by RikEskes** [comment]
Could this perhaps be related to #2189 ?

**Comment by trimox** [comment]
@RikEskes Yep, both are related issue. Does your StackBlitz ever finish compiling and starting dev server?

![image](https://user-images.githubusercontent.com/11887843/79552759-7751ea80-8069-11ea-8145-296a1a61a6d1.png)

**Comment by RikEskes** [comment]
The StackBlitz used to finish compiling. If it continues not to do so I will create a new one to ensure the issue is reproducible.

**Comment by trimox** [comment]
Wondering if enabling `Ivy` on StackBlitz is causing infinite compilation. Same thing occurs with https://stackblitz.com/edit/angular-mdc

**Comment by trimox** [comment]
Hrmm. Not sure this is an `@use` issue since `import` fails as well. Maybe something with the approach used inside `angular.json`'s `stylePreprocessorOptions` to help Sass compiler locate the MDC Web Sass files?

```
"stylePreprocessorOptions": {
              "includePaths": [
                "node_modules/"
              ]
            },
```

**Comment by RikEskes** [comment]
I created a new StackBlitz project which does finish compiling:
https://stackblitz.com/edit/angular-mdc-agnnan?file=src%2Fstyles%2Fglobal.scss

**Comment by guillaumegarcia13** [comment]
I noticed that in your example, @RikEskes, you have commented out the following:
```
// // Angular MDC
// @use '@angular-mdc/theme/material';
```

Any reason for doing so? (this is the oppposite of the Getting Started section)
